pkg/controllers/cloud/networkpolicy_controller.go

```diff
@@ -637,6 +664,17 @@ func (r *NetworkPolicyReconciler) SetupWithManager(mgr ctrl.Manager) error {
 				return sgs, nil
 			},
 		})
+	r.cloudRuleIndexer = cache.NewIndexer(
```

Comment:
Add a comment on what this cache is supposed to hold and used for

Reply:
Done